src/objc/TuriCreateObjC.h

```diff
@@ -1,3 +1,8 @@
+/* Copyright © 2017 Apple Inc. All rights reserved.
```

Comment:
The year should probably say 2018.

Reply:
Let's handle that in a separate PR. That needs to be fixed everywhere.